src/components/Utilities/UtilitiesTable.tsx

```diff
@@ -25,6 +25,15 @@ const criteriaMap = {
   created: FileSortCriteriaEnum.CreatedAt,
 };
 
+function isFileType(target: FileType | SeriesType): target is FileType {
+  // @ts-expect-error - Trying to access an unknown property.
+  return typeof target.ID === 'number';
+}
+
+function selectRowId(target: FileType | SeriesType): string | number {
+  return isFileType(target) ? target.ID : target.IDs.ID;
```

Comment:
why is `isFileType` a separate function when you could've just written that condition here? Also the type doesn't need to be checked. `target.ID ?? target.IDs.ID` would be fine.

Reply:
Try it and find out why tsc and eslint doesn't like it. 😃

Reply:
The `isFileType` is a type guard that ensures the target is a `FileType`. So tsc is happy since we narrowed down the type.